plugin-api/src/main/kotlin/io/mcarle/konvert/plugin/api/extensions.kt

```diff
@@ -16,3 +26,34 @@ inline fun <reified T : Annotation> T.extendProxy(): T {
         this
     }
 }
+
+@OptIn(KspExperimental::class, DelicateKotlinPoetApi::class)
+inline fun <reified T: Annotation, reified R: Annotation> passthroughAnnotation(
```

Comment:
Could you please add KDoc to this function so that it is clear, when you have to/should use this little helper?